bundles/org.openhab.binding.connectedcar/README.md

```diff
@@ -0,0 +1,585 @@
+|              | climater                    | Switch               | no      | Turn climatisation on/off                                                                       |
+|              | targetTemperature           | Number:Temperature   | yes     | Target temperature for the A/C climater                                                         |
+|              | targetChgLvl                | Number:Dimensionless | no      | CHarging stops automatically when the given level is reached                                    |
+|              | preHeater                   | Switch               | no      | Turn pre-heating on/off                                                                         |
+|              | duration                    | Number               | no      | Duration to run the ventilation/pre-heater in minutes                                           |
+|              | windowHeat                  | Switch               | no      | Turn window heating on/off                                                                      |
+|              | ventilation                 | Switch               | no      | Turn ventilation on/off                                                                         |
+|              | flash                       | Switch               | no      | ON: Triggers lights flashing                                                                    |
+|              | honkFlash                   | Switch               | no      | ON: Triggers Honk and Flash                                                                     |
+|              | hfDuration                  | Number               | no      | Duration in seconds for Flash/Honk &amp; Flash                                                  |
+|              | update                      | Switch               | no      | Force status update of vehicle status                                                           |
+| status       | odometer                    | Number:Length        | yes     | The  overall distance of the odometer when status was captured                                  |
+|              | carMoving                   | Switch               | yes     | ON: Car is moving                                                                               |
+|              | currentSpeed                | Number:Speed         | yes     | Current speed when data was last updated                                                        |
+|              | vehicleLocked               | Switch               | yes     | ON: Vehicle is completely locked. This includes doors, windows, but also hood and trunk         |
+|              | tempOutside                 | Number:Temperature   | yes     | The outside temperature in °C.                                                                  |
```

Comment:
If the user selected °F in the OH settings, this will automatically convert to °F.
```suggestion
|              | tempOutside                 | Number:Temperature   | yes     | The outside temperature                                                                  |
```